### PR TITLE
Bump Dependencies. Expo SDK 40 / React Native 0.63

### LINKED ISSUE #306: Upgrade to Expo SDK 39
Expo SDK 39 has tons of new features and improvements.
Also, SDK 36 support will be dropped next release.

Looks like its time to upgrade

### PR DESCRIPTION
**Pull request by artyorsh**
### Please read and mark the following check list before creating a pull request:

 - [x] I read and followed the [CONTRIBUTING.md](https://github.com/akveo/kittenTricks/blob/master/CONTRIBUTING.md) guide.

 #### Short description of what this resolves:
 
 Closes #306

### CODE REVIEW

**Review by malashkevich — APPROVED** [review]

**Comment by whitestranger7** [comment]
There's an issue with expo web loading. After `app-loading.component.expo` change, web load crashes with erorr:

Screenshot 2021-03-02 at 12 45 59

File: src/app/app-loading.component.expo.tsx [hunk 1]
```diff
@@ -19,70 +18,66 @@ export const LoadFontsTask = (fonts: { [key: string]: number }): Promise<TaskRes
...
  */
-export const AppLoading = (props: ApplicationLoaderProps): React.ReactElement => {
+export const AppLoading: React.FC<ApplicationLoaderProps> = (props) => {
 
   const [loading, setLoading] = React.useState<boolean>(true);
-  const loadingResult = props.initialConfig || {};
+  const loadingResult = React.useRef(props.initialConfig || {});
 
-  const onTasksFinish = (): void => {
+  const onTasksFinish = React.useCallback((): void => {
```
Comment (malashkevich) [inline src/app/app-loading.component.expo.tsx]: I don't think you need `useCallback ` here

File: src/app/app-loading.component.expo.tsx [hunk 2]
```diff
@@ -19,70 +18,66 @@ export const LoadFontsTask = (fonts: { [key: string]: number }): Promise<TaskRes
...
-  };
-
-  const startTasks = (): Promise<any> => {
-    if (props.tasks) {
-      return Promise.all(props.tasks.map(createRunnableTask));
-    }
-    return Promise.resolve();
+  const startTasks = (): Promise<void> => {
+    return Promise.all([
+      ...props.tasks.map(task => task().then(saveTaskResult)),
```
Comment (malashkevich) [inline src/app/app-loading.component.expo.tsx]: map already returns an array

File: tsconfig.json [hunk 3]
```diff
@@ -17,15 +17,7 @@
     "skipLibCheck": true,
     "typeRoots": [
       "./node_modules/@types"
-    ],
-    "paths": {
```
Comment (malashkevich) [inline tsconfig.json]: this block should stay I guess

**Review by malashkevich — CHANGES_REQUESTED** [review]
Please make sure kitten tricks works properly with local UI kitten

**Pull request merged**